loguru_mypy/__init__.py

```diff
@@ -73,6 +75,14 @@ def _loguru_logger_call_handler(
     except IndexError:
         call_kwargs = {}
 
+    call_expr = CallExpr(
+        callee=MemberExpr(expr=log_msg_expr, name="format"),
+        args=call_args,
+        arg_kinds=ctx.arg_kinds[1],
+        arg_names=ctx.arg_names[1],
```

Comment:
I'm not sure I understand mypy's structure there. Why are all those `arg_*` attributes lists of lists, and why is the second one the one I'm interested in? It seems it has to do with "formal arguments" but I'm not sure I know what that means.

Reply:
Neither do I, TBH I debugged the hell out of it when implementing for the first time. 
From what I remember, it has something to do with the fact that mypy does not use enums, instead they have integers. So the second is is basically about either positional or named arguments (can't remember which one is which). Anyway, I am pretty sure there is a constant somewhere saying that `1=KWARGS` and `0=ARGS` or vice-versa.

Does that help at all?

Reply:
Yeah I figured that up along the way :slightly_smiling_face: